### PR TITLE
Add checking displayKey for parent folder

### PR DESCRIPTION
**Pull request by Sergey-weber**
Added a check for the equality of the display_key with the key from input parameter

### CODE REVIEW

File: src/db/models/entry/index.ts [hunk 1]
```diff
@@ -392,6 +395,16 @@ class Entry extends Model {
                     .first();
```
Comment (imsitnikov) [inline src/db/models/entry/index.ts:395]: Looks like the single change what should be in this PR:
```
parentFolder = await Entry.query(TRX)
    .select(RETURN_COLUMNS)
    .join('revisions', 'entries.savedId', 'revisions.revId')
    .where({
        tenantId,
        displayKey: Utils.getParentFolderKey({keyFormatted: displayKey}), // <- here
        isDeleted: false,
    })
    .first();
```

File: src/db/models/entry/index.ts [hunk 2]
```diff
@@ -208,6 +210,8 @@ class Entry extends Model {
             initialPermissions,
         });
 
+        console.log('keyasd: ', key);
```
Comment (imsitnikov) [inline src/db/models/entry/index.ts:213]: ?

**Review by imsitnikov — CHANGES_REQUESTED** [review]

File: src/db/models/entry/index.ts (hunk 2 above)
Comment (Sergey-weber) [reply src/db/models/entry/index.ts]: removed

File: src/db/models/entry/index.ts (hunk 1 above)
Comment (Sergey-weber) [reply src/db/models/entry/index.ts:393]: yea, usually we use query with key, i don't know why, fixed it

**Review by jhoncool — APPROVED** [review]

**Review by imsitnikov — APPROVED** [review]

**Pull request merged**